src/components/UIElements/MediaEmbed.vue

```diff
@@ -318,8 +326,8 @@ export default {
         <object type="application/x-shockwave-flash" 
           width="${this.flashProps.width}" 
           height="${this.flashProps.height}" 
-          data="${this.$getResourceURL(this.url)}">
-            <param name='movie' value="${this.$getResourceURL(this.url)}"/>
+          data="${Resources.resolveAssetsProtocol(this.$getResourceURL(this.url))}">
```

Comment:
On electron, all assets:// links are redirected by the browser component, but the webapp doesn't have that luxury. Most assets are covered by an extra DOM filter pass, but this skips a step by resolving the assets protocol. Since this happens in the element, at render time, it should be safe.